augur/io/sequences.py

```diff
@@ -448,29 +448,41 @@ def _read_genbank(reference, feature_names):
     return features
 
 
-def validate_sequence_file(file: str):
-    """Ensure all sequence identifiers in the file are unique."""
-    with NamedTemporaryFile("w+", encoding="utf-8") as dup_num_file:
+def read_sequence_ids(file: str, error_on_duplicates = True):
+    """Get unique identifiers from a sequence file."""
+    unique = set()
+    duplicates = set()
+
+    with NamedTemporaryFile("w+") as temp_file:
         command = f"""
             {augur()} read-file {shquote(file)} |
-            {seqkit()} rmdup --dup-num-file {shquote(dup_num_file.name)} > /dev/null
+            {seqkit()} fx2tab --name > {shquote(temp_file.name)}
```

Comment:
Saw the `--name` and `--only-id` flags for [`seqkit seq`](https://bioinf.shenwei.me/seqkit/usage/#seq) and wondered if it's the same for `seqkit fx2tab`.

Seems like the output for `seqkit fx2tab --name` will include the full head from the FASTA
```
$ cat dup.fasta
>seq_1
atcg
>seq_1 dup
cgta
$ seqkit fx2tab --name dup.fasta
seq_1
seq_1 dup
```
Adding the `--only-id` flag gets what we want
```
$ seqkit fx2tab --name --only-id dup.fasta
seq_1
seq_1
```